Quaver.Shared/Screens/Gameplay/UI/PauseScreen.cs

```diff
@@ -150,25 +156,65 @@ public PauseScreen(GameplayScreen screen)
             };
             Buttons.Add(Quit);
             Quit.Size = new ScalableVector2(Quit.Image.Width, Quit.Image.Height);
-            Quit.Hovered += (o, e) => HoverButton(Quit);
+            Quit.Hovered += (o, e) => HoverButton(2);
 
             // Select continue button on initialization
-            Selected = Continue;
-            HoverButton(Selected);
+            HoverButton(0, true);
+        }
+
+        private void HandleKeyPressUp()
+        {
+            if (!KeyboardManager.IsUniqueKeyPress(Keys.Up))
+                return;
+
+            var index = SelectedIndex == 0 ? Buttons.Count - 1 : SelectedIndex - 1;
+            HoverButton(index);
+        }
+
+        private void HandleKeyPressDown()
+        {
+            if (!KeyboardManager.IsUniqueKeyPress(Keys.Down))
+                return;
+
+            var index = SelectedIndex == Buttons.Count - 1 ? 0 : SelectedIndex + 1;
+            HoverButton(index);
+        }
+
+        private void HandleKeyPressSelect()
+        {
+            if (!KeyboardManager.IsUniqueKeyPress(Keys.Enter))
+                return;
+
+            // Call specific method for button selected
+            if (Selected.Equals(Continue))
+                InitiateContinue();
+
+            else if (Selected.Equals(Retry))
+                InitiateRetry();
+
+            else if (Selected.Equals(Quit))
+                InitiateQuit();
         }
 
         /// <summary>
-        ///     Called when a button gets hovered over
+        ///     Hover over a specific button via given index.
         /// </summary>
         /// <param name="button"></param>
-        private void HoverButton(ImageButton button)
+        private void HoverButton(int index, bool dontPlayAudio = false)
         {
-            Selected = button;
+            SelectedIndex = index;
+            Selected = Buttons[index];
             Buttons.ForEach(x => ClearNonAlphaAnimations(x));
 
+            // Play sfx
```

Comment:
If it's not implemented, remove the comments and open an issue.

Reply:
This should be addressed once a new design has been created for the overhaul. It's not worth the effort implementing feedback for this yet, and we'd have to create an issue for other menus/buttons as well.